core-services/image-mirroring/_config.yaml

```diff
@@ -555,6 +555,8 @@ supplementalCIImages:
     image: quay.io/stackrox-io/apollo-ci:scanner-test-0.3.66
   stackrox/apollo-ci:scanner-test-0.3.68:
     image: quay.io/stackrox-io/apollo-ci:scanner-test-0.3.68
+  stackrox/apollo-ci:scanner-test-0.3.69:
+    image: quay.io/stackrox-io/apollo-ci:scanner-test-0.3.69
```

Comment:
Ah, totally forgot this mirroring thing in OCI.

Reply:
![image](https://github.com/openshift/release/assets/7704495/a0456f54-ff2e-4997-8665-e5b70ccbba06)